deployment/ccip/operation/evm/v1_6/ops_fee_quoter.go

```diff
@@ -46,63 +43,39 @@ var (
 	DeployFeeQuoterOp = opsutil.NewEVMDeployOperation(
 		"DeployFeeQuoter",
 		semver.MustParse("1.0.0"),
-		"Deploys FeeQuoter 1.6 contract on the specified evm chain",
-		cldf.NewTypeAndVersion(shared.FeeQuoter, deployment.Version1_6_0),
-		opsutil.VMDeployers[DeployFeeQInput]{
-			DeployEVM: func(opts *bind.TransactOpts, backend bind.ContractBackend, input DeployFeeQInput) (common.Address, *types.Transaction, error) {
-				// TODO update once latest fee quoter is supported https://smartcontract-it.atlassian.net/browse/CCIP-7181?atlOrigin=eyJpIjoiODlmNDc2MDhmNTAyNGY1YmFhNGU3YTk3NGQ5ODk5ZjciLCJwIjoiaiJ9
-				// TEMP: Use latest fee quoter to support TON
-				// fee_quoter.FeeQuoterBin = latest_fee_quoter.FeeQuoterBin
-
-				addr, tx, _, err := fee_quoter.DeployFeeQuoter(opts, backend,
-					fee_quoter.FeeQuoterStaticConfig{
-						MaxFeeJuelsPerMsg:            input.Params.MaxFeeJuelsPerMsg,
-						LinkToken:                    input.LinkAddr,
-						TokenPriceStalenessThreshold: input.Params.TokenPriceStalenessThreshold,
+		"Deploys FeeQuoter 1.6.x contract on the specified evm chain",
```

Comment:
The version is hardcoded to 1.6.0 input below, should this description be changed?